packages/forms-web-app/src/controllers/householder-planning/eligibility/use-existing-service-costs.js

```diff
@@ -0,0 +1,13 @@
+const {
+  VIEW: {
+    HOUSEHOLDER_PLANNING: {
+      ELIGIBILITY: { USE_EXISTING_SERVICE_COSTS: useExistingServiceCosts },
+    },
+  },
+} = require('../../../lib/householder-planning/views');
+
+exports.getUseExistingServiceCosts = async (_, res) => {
+  res.render(useExistingServiceCosts, {
```

Comment:
in other controllers, the method inside render is called via the VIEW constant - is it worth changing this here to be consistent?

Reply:
In claim costs, date decision and decision date controllers under Householder Planning, the same methodology as here is being used. Refactoring other controllers in the forms-web-app service, such as full-appeal etc., means that there should be a single way of importing those constants throughout the controllers, and the team should agree on what that should be first.